### DISCUSSION TITLE
Issue with deployment of Oxalis version 6.5.0 on Tomcat version 10.0.27 with Java 11

### DISCUSSION THREAD

**Discussion by kirti2001** [OxalisCommunity/oxalis#674] [Q&A]
Hi,

We are facing issue with deployment of Oxalis version 6.5.0 on Tomcat version 10.0.27 with Java 11 on Windows 2016 server
Same Oxalis version 6.5.0 is working with Tomcat  version 9 with Java 11 on Windows 2016 server

In logs we are not finding anything specific. Anyone faced similar issue with Tomcat 10.

Any help/guidance is appreciated.

Thanks
Kirti

**Comment by dladlk** [comment]
Just use Tomcat 9. Tomcat 10 works with Servlet spec 6.0, Tomcat 9 - 4.0, there are differences in web app registration - including the change in package name from javax.* to jakarta.* (in spec 5.0)...
Oxalis current war build requires Tomcat 9 as it is built with javax.*

**Reply by kirti2001** [reply]
Hi,

Thanks for the reply.
Do you have tomcat10 on roadmap?

Thanks,
Kirti
